src/vscode-dts/vscode.d.ts

```diff
@@ -19046,7 +19046,29 @@ declare module 'vscode' {
 		/**
 		 * The value of this reference. The `string | Uri | Location` types are used today, but this could expand in the future.
 		 */
-		readonly value: string | Uri | Location | unknown;
+		readonly value: string | Uri | Location | ChatReferenceBinaryData | unknown;
+	}
+
+	/**
+	 * Represents binary data associated with a chat reference.
+	 */
+	export class ChatReferenceBinaryData {
+		/**
+		 * The MIME type of the binary data.
+		 */
+		readonly mimeType: string;
+
+		/*
+		* The binary data of the reference.
+		*/
+		readonly data: Uint8Array;
+
+		/**
+		 * Creates a new instance of ChatReferenceBinaryData.
+		 * @param mimeType The MIME type of the binary data.
+		 * @param data The binary data of the reference.
+		 */
+		constructor(mimeType: string, data: Uint8Array);
```

Comment:
Please make sure this does not get merged until it goes through api review too. Is it possible to ship as a proposal instead?